<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>687</width>
    <height>420</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>truck.png</normaloff>truck.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="vertical_layout_2">
    <property name="spacing">
     <number>5</number>
    </property>
    <property name="leftMargin">
     <number>5</number>
    </property>
    <property name="topMargin">
     <number>5</number>
    </property>
    <property name="rightMargin">
     <number>5</number>
    </property>
    <property name="bottomMargin">
     <number>5</number>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontal_layout">
      <item>
       <widget class="QPushButton" name="button_search">
        <property name="text">
         <string>Найти</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="button_remove">
        <property name="text">
         <string>Удалить </string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontal_spacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="list_widget_junk"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="vertical_layout">
      <item>
       <widget class="QProgressBar" name="progress_bar">
        <property name="value">
         <number>24</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menu_bar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>687</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="action_connect"/>
    <addaction name="action_disconnect"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="action_connect">
   <property name="text">
    <string>Подключиться</string>
   </property>
  </action>
  <action name="action_disconnect">
   <property name="text">
    <string>Отключиться</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
